_templates/day/new/day.ejs.t

```diff
@@ -0,0 +1,34 @@
+---
+to: <%= year %>/day-<%= day %>.spec.js
+---
+
+const {
+  part1,
+  part2,
+  data,
+} = require("./day-<%= day %>");
+
+describe("Day <%= day %>", () => {
+  describe("Part 1", () => {
+    it("works for test data", () => {
+      expect(part1(data("test"))).toBe(undefined);
+    });
+
+    it("works for real data", () => {
```

Comment:
One improvement I've actually made is to run these in the same test suite, like:

```js
// Simple test data
// expect(part1(data([]))).toBe(undefined);

// Test data
expect(part1(data("test"))).toBe(undefined);

// Real data
expect(part1(data))).toBe(undefined);
```

This way during the early phases, I have less test-spam to deal with.

Reply:
That makes sense! i ended up using this cute little run button alot to run only the specific one I wanted
![image](https://user-images.githubusercontent.com/38425820/145227444-e141fb5d-28a6-4b0f-b6bf-d76ad8df04c6.png)

Reply:
(i should really give that debug button a try sometime...)

Reply:
Protip: open a terminal next to  your IDE and use `yarn --watch` (maybe VS Code can do this; I dunno)